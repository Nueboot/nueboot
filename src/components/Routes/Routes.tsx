import { Router } from '@reach/router';
import React from 'react';

import Home from '../Home';

const App: React.SFC = () => (
  <Router>
    <Home default path="/" />
  </Router>
);

export default App;
