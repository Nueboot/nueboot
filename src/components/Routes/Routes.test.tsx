import { shallow, ShallowWrapper } from 'enzyme';
import React from 'react';

import Routes from './Routes';

describe('Routes', () => {
  let component: ShallowWrapper;

  beforeEach(() => {
    component = shallow(<Routes />);
  });

  it('renders the right routes', () => {
    expect(component).toMatchSnapshot();
  });
});
