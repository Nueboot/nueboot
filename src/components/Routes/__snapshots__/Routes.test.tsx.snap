// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Routes renders the right routes 1`] = `
<Router>
  <Home
    default={true}
    path="/"
  />
</Router>
`;
